src/renderer/modules/Settings/GeneralSettings.js

```diff
@@ -72,6 +72,7 @@ export default class GeneralSettings extends Component {
       spanishF,
       germanF,
       frenchF,
+      frenchF,
```

Comment:
Shouldn’t it be something like frenchBepo instead ?

Reply:
It does not matter, the flag is the same, the French one. So… If we can avoid duplication.